toffy/bin_extraction.py

```diff
@@ -28,6 +28,10 @@ def extract_missing_fovs(bin_file_dir, extraction_dir, panel,
     if empty_fovs:
         fovs = list(set(fovs).difference(empty_fovs))
 
+    if len(fovs) == 0:
+        warnings.warn(f"No viable bin files were found in {bin_file_dir}", Warning)
```

Comment:
Let's add a test to make sure this works as expected. It's very simple, so if we were starting from scratch it wouldn't be worth having a test. However, we should always add a reversion test when fixing bugs to make sure they don't come back.